tests/models/vits/test_modeling_vits.py

```diff
@@ -177,6 +179,30 @@ def test_model_forward(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_model_forward(*config_and_inputs)
 
+    @require_torch_multi_gpu
+    # override to make test deterministic across GPUs
```

Comment:
maybe more informative: `make all elements of the batch have the same length` (despite redundant below)

(unclear to me what `deterministic` means here)

Reply:
Resolved in https://github.com/huggingface/transformers/pull/25986/commits/63a592106f97801c1b2e1e46aa246123487e45f6